src/test/ui/traits/issue-65673.rs

```diff
@@ -7,6 +7,6 @@ trait Alias<T> = where T: Trait;
 
 impl<T> WithType for T {
     type Ctx = dyn Alias<T>;
-//~^ ERROR the size for values of type `(dyn Trait + 'static)` cannot be known at compilation time
+    //~^ ERROR at least one trait is required for an object type [E0224]
```

Comment:
@compiler-errors Is this not a diagnostics regression?

Reply:
why would this be a diagnostics regression? i thought the previous error message was nonsense, since trait alias "Alias" has no trait types...

Reply:
I think you're right, this isn't a regression. The new error message is somewhat confusing though.